test/e2e/wireguard_test.go

```diff
@@ -103,6 +104,63 @@ func testPodConnectivity(t *testing.T, data *TestData) {
 	defer deletePods()
 	numPods := 2
 	data.runPingMesh(t, podInfos[:numPods], agnhostContainerName)
+
+	// Make sure that route to Pod on peer Node and route to peer gateway is targeting the WireGuard device.
+	srcPod, err := data.getAntreaPodOnNode(nodeName(0))
+	require.NoError(t, err)
+	var srcIP, peerGatewayIP, peerPodIP string
+	IPv4, IPv6 := nodeGatewayIPs(0)
```

Comment:
ipv4, ipv6

local variables shouldn't be capitalized

Reply:
done, will keep in mind.